sounds/views.py

```diff
@@ -244,6 +245,16 @@ def sound(request, username, sound_id):
     is_explicit = sound.is_explicit and (not request.user.is_authenticated() \
                         or not request.user.profile.is_adult)
 
+    # Show modal asking users for donations after clicking in download
```

Comment:
This is kind of messy... keep the templating stuff inside the templates. You are doing string-insertion by python string-mashing. That's quite bad...